OpenEdXMobile/res/values/strings.xml

```diff
@@ -482,7 +482,7 @@
 
     <!-- Registration Screen -->
     <!-- Create account agreement. Followed by list of agreement names like "End User License Agreement" -->
-    <string name="by_creating_account">By creating an account, you agree to the </string>
+    <string name="by_creating_account">By creating an {platform_name} account, you agree to the </string>
```

Comment:
@mdinino How about this wording, to get rid of the agreement issue?
"By creating an account on ```(platform_name}```, you agree to the ..."